test/helper/launcher/wait.go

```diff
@@ -0,0 +1,28 @@
+package launcher
+
+import (
+	"fmt"
+	"time"
+)
+
+var (
+	NoWait = (*WaitConfig)(nil)
+)
+
+type WaitConfig struct {
+	Condition string
+	Target    string
+	Timeout   time.Duration
+}
+
+func WaitReady(target string, timeout time.Duration) *WaitConfig {
+	return &WaitConfig{Condition: "condition=Ready", Target: target, Timeout: timeout}
+}
+
+func (cfg *WaitConfig) waitArgs() []string {
+	args := []string{"wait", fmt.Sprintf("--for=%s", cfg.Condition), cfg.Target}
+	if cfg.Timeout == 0 {
```

Comment:
It's a mistake or do you really want to set timeout only when it's zero?

Reply:
This was a mistake, the logic had to be reversed